### DISCUSSION TITLE
Right arrow key binding for context menu

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#715] [Ideas]
![image](https://user-images.githubusercontent.com/6903107/135309582-e58b063b-32c0-42eb-b87a-a65da96d6002.png)
How about this function? it is open context menu when you press the right arrow at the end of the query box. this keybinding some launcher use. (listary, fluent launcher) but it's not in the flow launcher. I've been using it for a some hours. it's good. I often use folder open with files and it's useful because my hand is on the arrow key. Exceptions already been made if no query situation or or if you press right in the context menu. I don't want to dirty the code, so I want someone to put it in.

**Comment by jjw24** [comment]
yeah good idea. right arrow at the end of query box to go into the context menu and left arrow at the start of context menu to get out of it (saves from pressing esc)?

**Reply by onesounds** [reply]
yeah I was made left arrow feature and erased because of the case of moving the query cursor. (Listary don't use left, Fluent uses left.) Come to think of it, there are few additional queries in the context menu, so I think we can use left.
